msos/StackWalker.cs

```diff
@@ -99,7 +111,11 @@ abstract class StackWalkerStrategy
         public const string WAIT_FOR_MULTIPLE_OBJECTS_FUNCTION_NAME = "WaitForMultipleObjects";
         public const string WAIT_FOR_MULTIPLE_OBJECTS_EX_FUNCTION_NAME = "WaitForMultipleObjectsEx";
         public const string ENTER_CRITICAL_SECTION_FUNCTION_NAME = "RtlEnterCriticalSection";
+        public const string THREAD_SLEEP_FUNCTION_NAME = "Thread.Sleep";
+        public const string SLEEP_FUNCTION_NAME = "SleepEx";
```

Comment:
Is this supposed to be `Sleep`? I think it might be easier to detect all flavors of sleep by looking for `NtDelayExecution` (from ntdll). Worth checking out.